index.html

```diff
@@ -56,10 +56,10 @@ <h2 >
         The home of wallpepers
     </h2>
     <div class="category">
-      <button id="abstract-btn" type="submit">Abstract</button>
-      <button id="nature-btn" type="submit">Nature</button>
-      <button id="cosmo-btn" type="submit">Cosmo</button>
-      <button id="minimal-btn" type="submit">Minimal</button>
+      <button id="abstract-btn" name="Abstract" type="submit">Abstract</button>
+      <button id="nature-btn" name="Nature" type="submit">Nature</button>
+      <button id="cosmo-btn" name="Cosmo" type="submit">Cosmo</button>
+      <button id="minimal-btn"  name="Minimal" type="submit">Minimal</button>
```

Comment:
There is no need to add name attribute by the button content in order to get it in JS
You can access it directly by this code `event.target.textContent`